src/ui/public/chrome/chrome.html

```diff
@@ -80,7 +64,7 @@
         </div>
         <!-- /Full navbar -->
       </nav>
-      <div class="application" ng-class="'tab-' + chrome.getFirstPathSegment() + ' ' + chrome.getApplicationClasses()" ng-view></div>
+      <div class="application" ng-class="'tab-' + chrome.getFirstPathSegment() + ' ' + chrome.getApplicationClasses()" ng-view kbn-loading-indicator></div>
```

Comment:
The loading indicator is absolute positioned anyway, not sure if you need the complexity you have in place to keep it an attr instead of an element, and the testing to make sure it doesn't modify existing content. Thoughts?